packages/zoe/src/contractFacet/zcfZygote.js

```diff
@@ -379,12 +380,17 @@ export const makeZCFZygote = async (
           publicFacet = undefined,
           creatorInvitation = undefined,
         }) => {
-          const allDurable = [
-            creatorFacet,
-            publicFacet,
-            creatorInvitation,
-          ].every(canBeDurable);
-          if (prepare || allDurable) {
+          const areDurable = objectMap(
+            { creatorFacet, publicFacet, creatorInvitation },
+            canBeDurable,
+          );
+          const allDurable = Object.values(areDurable).every(Boolean);
```

Comment:
why do you prefer this version?  It seems to make two passes through the list (admittedly small) to get the same result.

Reply:
the status quo is to just say something wasn't durable. this way it tells you which weren't (by keeping track of `areDurable`)